x-pack/plugins/elastic_assistant/server/ai_assistant_service/index.ts

```diff
@@ -124,6 +160,36 @@ export class AIAssistantService {
         pluginStop$: this.options.pluginStop$,
       });
 
+      if (this.initializeKnowledgeBase) {
+        await this.knowledgeBaseDataStream.install({
+          esClient,
+          logger: this.options.logger,
+          pluginStop$: this.options.pluginStop$,
+        });
+
+        // TODO: Pipeline creation is temporary as we'll be moving to semantic_text field once available in ES
+        const pipelineCreated = await pipelineExists({
+          esClient,
+          id: this.resourceNames.pipelines.knowledgeBase,
+        });
+        if (!pipelineCreated) {
+          this.options.logger.debug(
+            `Installing ingest pipeline - ${this.resourceNames.pipelines.knowledgeBase}`
+          );
+          const response = await createPipeline({
```

Comment:
As noted in the code comments, this call to `createPipeline` may be removed in the future, but for now consider catching any errors thrown here, to ensure the other data streams are still initialized.

Reply:
Added a try/catch in [00468fa](https://github.com/elastic/kibana/pull/182763/commits/00468fa5cc0537d166f69b1135185654838e3cd0)